README.md

```diff
@@ -42,6 +42,16 @@ We will only reduce the fan level again once:
 
 This avoids unnecessary fluctuations in fan speed.
 
+### Watchdog
+
+In case of zcfan crashing and not recovering (e.g. restart via systemd),
```

Comment:
This software is for power users, just mentioning how to adjust it is fine without a lot of explanation about how the watchdog works.

Reply:
Agreed, text modified.